src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/jpa/entity/WatchlistEntity.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.CascadeType
+
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "watchlist")
+class WatchlistEntity(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long?,
+
+  @ManyToOne(cascade = [CascadeType.MERGE])
+  @JoinColumn(name = "prisoner_id", referencedColumnName = "id")
+  val prisoner: PrisonerEntity,
+
+  @Column(name = "staff_username")
+  var staffUsername: String?,
```

Comment:
I think you can make this final and non-nullable
```suggestion
  val staffUsername: String,
```